README.rst

```diff
@@ -115,3 +123,4 @@ Links
 - `Documentation <https://docs.nextcord.dev/>`_
 - `Official Discord Server <https://discord.gg/ZebatWssCB>`_
 - `Discord API <https://discord.gg/discord-api>`_
+- `Nextcord's Github <https://www.github.com/nextcrd/nextcord>`_
```

Comment:
This is spelled wrong, but also, what's the point of including the GitHub link in the GitHub readme? You're already on the page when you're looking at it.

If it's for Pypi, it's also not needed since there is already a "Homepage" and "Issue tracker" link on Pypi that links here.

Reply:
Just for the nextcord readthedocs page.

Reply:
thats a completely different file entirely

Reply:
Removed, thanks